tests/test_black.py

```diff
@@ -359,6 +359,14 @@ def test_string_quotes(self) -> None:
         black.assert_equivalent(source, not_normalized)
         black.assert_stable(source, not_normalized, mode=mode)
 
+    @patch("black.dump_to_file", dump_to_stderr)
+    def test_multiline_strings(self) -> None:
+        source, expected = read_data("multiline_strings")
+        actual = fs(source)
+        self.assertFormatEqual(expected, actual)
+        black.assert_equivalent(source, actual)
+        black.assert_stable(source, actual, DEFAULT_MODE)
+
```

Comment:
Since PR #1785, writing simple tests like this one is quite easier. Just make the test data and add its normalized name (i.e. strip the `.py` suffix) in the SIMPLE cases list in `tests/test_format.py`.

[I've done this for you in a PR](https://github.com/lyft/black/pull/1) against your branch since I can't suggest changes on lines/files you didn't modify yet I need to suggest a single addition in such a file :/